examples/extensions/r2/hello_r2.py

```diff
@@ -35,6 +35,7 @@ def my_sandbox(path, rootfs):
     ql.hook_address(func, r2.functions['main'].offset)
     # enable trace powered by r2 symsmap
     # r2.enable_trace()
+    r2.set_backtrace(0x401906)
     ql.run()
 
 if __name__ == "__main__":
```

Comment:
It looks like this file contains two differnt examples.
Please break them down to two files, or clarify how they are related to each other (in case trhey are).